probes/notCreatedInLast90Days/def.yml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023 OpenSSF Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+id: notCreatedInLast90Days
```

Comment:
in general: do we want to avoid using negative probe names? When writing policies, that requires doing double negations which seem harder for human to get right. Wdut?

Reply:
We have discussed it previously, and one of the points that were raised was that the opposite is also confusing. In this case, it is positive if the project is created more than 90 days ago. `OutcomePositive` should reflect both "yes" to the question in the probe name as well as the part Scorecard considers positive. Therefore, in order to answer "yes"/`OutcomePositive`, the probe has a name that allows us to answer as such.

A different name that would convey the same is `createdMoreThan90DaysAgo`, where "yes" and `OutcomePositive` align.

Reply:
do you have a link to the previous discussion? 

> OutcomePositive should reflect both "yes" to the question in the probe name as well as the part Scorecard considers positive.

ah, I see what you mean. Basically our remediation section assumes that the probe's name is a positive thing. It would not make sense to have:
```yml
id: createdRecently
remediation:
  text: The only remediation for this probe is to wait until 90 days have passed after a project has been created.
```

Does this mean we're effectively biased in what our probes do, and someone who disagrees (say, they create their own check with `notCreatedRecently is false` - whereas scorecard's built-in check) would receive the wrong remediation?

What I mean is that the only way to use the probes is to combine them "positively", e.g. `probeA is true and probeB is true` or `probeA is true or probeB is true`, but `probeA is false and / or probeB is false` would not work? I think this means that users can't negate our probes, they can only _ignore_ them.

Are their cases where this is bad?

Reply:
Also resolving this, tracking in #3654 instead